charts/nopo11y/README.md

```diff
@@ -22,25 +23,57 @@ To install the chart:
-| prometheusReleaseLabel            | string | no        | nopo11y-stack  |                                                                                    |
-| availabilitySLO                   | float  | no        | 99.9           | Threshold for service availability SLO. Value is in percentage                     |
-| latencySLO                        | float  | no        | 99             | Threshold for service latency SLO. Value is in percentage                          |
-| latency                           | number | no        | 1000           | Threshold for overall api latency alert. Value is in milliseconds                  |
-| errorRate5xx                      | float  | no        | 0.05           | Threshold for 5xx error rate alert. Value is in percentage                         |
-| errorRate4xx                      | float  | no        | 5              | Threshold for 4xx error rate alert. Value is in percentage                         |
-| grafanaURL                        | string | no        | <empty>        | External grafana url, if set then dashboard links will be visible in the alert     |
-| logLabel                          | string | no        | <empty>        | If set then logs will be filtered with specified log label in logs dashboard panel |
-| logLabelValue                     | string | no        | <empty>        | value for the logLabel                                                             |
-| istioMetrics.enabled              | bool   | no        | true           | If true then istio metrics will be visible in prometheus                           |
-| nginxIngressMetrics.enabled       | bool   | no        | false          | If true then nginx ingress metrics will be visible in prometheus                   |
-| nginxIngressMetrics.ingressName   | string | no        | sample-ingress | Name of the ingress                                                                |
+| defaults                         | map | no       | -      | Default values for SLO objective and alert thresholds for all services                                           |
+| prependReleaseName | bool   | no        | false          | If set to true then it will prepend release name to service and deployment name                       |
+| grafanaURL            | string | no        | empty  |  External grafana URL                                                                       |
+| apiGateway | string | yes | istio | Which api gateway metrics to use, allowed values istio, nginx |
+| services | array | yes | - | list of services for which you want to create dashboard, alerts and SLOs |
+
+# Sample values
+```yaml
+enabled: false
+
+defaults:
+  slo:
+    availability: 99.9
+    latency: 99
+  alertThresholds:
+    latencyMS: 100
+    rate5xx: 0.05
+    rate4xx: 5
+
+namespace: observability
+
+prependReleaseName: false
```

Comment:
Is this flag needed anymore? Since we're decoupling it from ODA.

Reply:
As discussed with @shehbaz-pathan this will be removed in the subsequent release.